roles/gitea-ocp/tasks/main.yml

```diff
@@ -52,6 +52,15 @@
   set_fact:
     _gitea_actual_route: "{{ r_route.resources[0].spec.host }}"
 
+- name: Get external hostname from Service if external SSH is needed
+  k8s_info:
+    api_version: v1
+    kind: Service
+    name: "{{ _gitea_name }}"
+    namespace: "{{ _gitea_namespace }}"
+  register: gitea_service
+  when: _gitea_load_balancer
```

Comment:
Might be good to `block:` this when and add a `set_fact` to drop the the ingress url for the load balancer, to make the template for the configmap less likely to fail